gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-client/src/main/java/org/apache/gobblin/service/FlowConfigV2Client.java

```diff
@@ -256,6 +257,19 @@ public void deleteFlowConfigWithStateStore(FlowId flowId)
     response.getResponse();
   }
 
+  public String triggerFlowConfig(FlowId flowId)
```

Comment:
Should this also be updated to reflect a new name? Also in other places.

Reply:
Yeah it should have been, updated that as well.